modules/assim.batch/R/pda.emulator.R

```diff
@@ -12,22 +10,23 @@
 ##' @author Ryan Kelly, Istem Fer
 ##' @export
 pda.emulator <- function(settings, params.id=NULL, param.names=NULL, prior.id=NULL, chain=NULL, 
-                     iter=NULL, adapt=NULL, adj.min=NULL, ar.target=NULL, jvar=NULL, n.knot=NULL) {
+                     iter=NULL, adapt=NULL, adj.min=NULL, ar.target=NULL, jvar=NULL, n.knot=NULL, burnin=NULL) {
   
   ## this bit of code is useful for defining the variables passed to this function 
   ## if you are debugging
   if(FALSE){
     params.id <- param.names <- prior.id <- chain <- iter <- NULL 
-    n.knot <- adapt <- adj.min <- ar.target <- jvar <- NULL
+    n.knot <- adapt <- adj.min <- ar.target <- jvar <- burnin <- NULL
   }
 
   ## -------------------------------------- Setup ------------------------------------- ##
   ## Handle settings
     settings <- pda.settings(
                   settings=settings, params.id=params.id, param.names=param.names, 
                   prior.id=prior.id, chain=chain, iter=iter, adapt=adapt, 
-                  adj.min=adj.min, ar.target=ar.target, jvar=jvar, n.knot=n.knot)
+                  adj.min=adj.min, ar.target=ar.target, jvar=jvar, n.knot=n.knot, burnin=burnin)
 
+   burnin <- ifelse(!is.null(settings$assim.batch$burnin), settings$assim.batch$burnin, ceiling(min(2000,0.2*settings$assim.batch$iter)))
```

Comment:
Just a formatting suggestion when calling functions with many/long arguments. Go to a new line after every argument. It makes it a little easier to read.
```
burnin <- ifelse(!is.null(settings$assim.batch$burnin), 
                          settings$assim.batch$burnin, 
                          ceiling(min(2000,0.2*settings$assim.batch$iter))
                ) 
```